examples/java/src/main/java/AppendObject.java

```diff
@@ -0,0 +1,89 @@
+import com.amazonaws.AmazonServiceException;
+import com.amazonaws.client.builder.AwsClientBuilder;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.AmazonS3Exception;
+import com.amazonaws.services.s3.model.Bucket;
+import com.amazonaws.services.s3.model.ListObjectsV2Result;
+import com.amazonaws.services.s3.model.S3ObjectSummary;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+public class AppendObject {
+
+    public static void uploadObject(String object_name, String bucket_name, String object_path, AmazonS3 s3) {
+        try {
+            s3.putObject(bucket_name, object_name, new File(object_path));
+            System.out.printf("Successfully uploaded object %s to bucket %s.\n", object_name, bucket_name);
+        } catch (AmazonServiceException e) {
+            System.err.println(e.getErrorMessage());
+            System.exit(1);
+        }
+    }
+
+    // This is a utility function to help create a bucket if not existed.
```

Comment:
why do you need to check if the bucket already exists before you create it?

Reply:
also, why do you first look for it manually in the list of buckets, and then use the API to check if it exists?

Reply:
Thanks very much for the comments!

For the first question, I suppose that the user may already create a bucket named mybucket, and if the bucket already exists, we cannot create a bucket with the same name, which will show an error message and stop the following procedure (uploading a bucket). Therefore, I need to check before creating a bucket. It seems there is no such problem in Golang SDK. In Golang SDK, if there exists a bucket with the same name, it will proceed with the following procedures. I need to further check this.

For the second question, I forgot to remove the manually listing part when I learned the AWS Java SDK. I will remove it in the modification.